default/stringtables/en.txt

```diff
@@ -1015,8 +1015,14 @@ Your connection was replaced.
 ERROR_NONPLAYER_CANNOT_CONCEDE
 Only Players can concede.
 
-ERROR_CONCEDE_NOT_ALLOWED
-Concede is not allowed.
+ERROR_CONCEDE_DISABLED
+Concede is disabled.
+
+ERROR_CONCEDE_EXCEED_COLONIES
+Your colonies number exceeds limits to concede.
```

Comment:
Don't refer to the player as "you". Instead maybe "Cannot concede; empire controls too many colonies."

Reply:
Fixed.